src/commands/commerce/init.js

```diff
@@ -81,14 +85,35 @@ export class InitCommand extends Command {
     console.log(`${boldWhite}Run locally:${reset} "aio commerce:dev"`)
     console.log('For next steps, including how to customize your storefront and make it your own, check out our docs:\nhttps://experienceleague.adobe.com/developer/commerce/storefront/')
 
-    // if we created a mesh, wait for verification to complete before exiting
-    // TODO: Replace with detached childProcess.
     if (shouldCreateMesh) {
-      await checkAndRetryMeshUpdate(runAIOCommand)
+      // Spawn detached child process to verify mesh in the background, without disrupting user's CLI session.
```

Comment:
So this will kick off a detached child process and will log to `mesh-verify.log`. I'm a little concerned that it could fail and the user would not know. Based on my findings in node docs, If we want the parent process to exit before this, we cannot log to the terminal.

Perhaps we should add a log here, before we kick off the command, like `Verifying Mesh Creation. Please check mesh-verify.log for details, or run "aio commerce:mesh-verify" if there are failures`.

Reply:
And actually, I don't think this is working. When I run through the CLI for the first time, and select my project/workspace and "use for future operations", the mesh-verify command is still asking for user to select project/workspace.

<img width="1212" alt="image" src="https://github.com/user-attachments/assets/7ff42cae-b6b5-4d15-b493-091b1910eaf3" />

<img width="681" alt="image" src="https://github.com/user-attachments/assets/e6822d20-ad6d-4072-bb16-5a0324d9d0b2" />

Reply:
Yeah probably because you cleared the console or config. Ideally that shouldn't be the case. Ill verify it.

Reply:
I did not though - unless I missed it programatically. I ran the cli, which would have had it set after picking mesh, and then immediately after, the mesh-verify command runs.

Reply:
Fixed the `mesh-verify` issue in the past few commits. Also with d2e06e1 we can run `mesh-verify` command with few optional flags like this `aio commerce mesh-verify --orgId 233240 --projectId 4566206088344816749 --workspaceId 4566206088344817114`